app/views/contexts/summary/group.py

```diff
@@ -77,10 +93,48 @@ def _build_blocks(
                     ]
                 )
 
+            elif block["type"] == "ListCollector":
+                section = schema.get_section(location.section_id)
+
+                if summary_item := schema.get_summary_item_for_list_for_section(
```

Comment:
Minor, this is 4 levels of nesting. Inverting would improve readability. (Non blocker)